ls1intum/Artemis
# [Feature] Grading System - Participant Scores

### Checklist

### Motivation and Context


This PR integrates the grading system with the participant scores pages for courses and exams

### Description

I've added an average grade and average rated grade on top of the participant scores pages.
In addition, the average scores table now also has an average grade and average rated grade columns.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> Grading Key
3. Create a Grading Key
4. Navigate to Participant scores and verify that (the course should have at least one exercise submission)
a) an average grade and an average rated grade are displayed at the top of the page
b) in the average scores table to last two columns display an average grade and an average rated grade
5. Create a Grading Key for an exam
6. Go through step 4 again for the exam (the exam should have at least one submission)

### Test Coverage



unchanged (100% for both participant scores components)

### Screenshots


![Screenshot 2021-07-09 143600](https://user-images.githubusercontent.com/63168603/125072248-0742e080-e0c3-11eb-9d9b-58c4dd9cea97.png)
